<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>36</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>JID:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineJid"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="linePassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushSignIn">
           <property name="text">
            <string>Sign In</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>36</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Based on &lt;a href=&quot;http://xmpp.org&quot;&gt;XMPP&lt;/a&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelStatus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabs">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>203</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>156</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushAddTab"/>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>156</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>203</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineJid</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>pushSignIn</tabstop>
  <tabstop>pushCancel</tabstop>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushSignIn</sender>
   <signal>clicked()</signal>
   <receiver>ChatWindow</receiver>
   <slot>slotSignIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushCancel</sender>
   <signal>clicked()</signal>
   <receiver>ChatWindow</receiver>
   <slot>slotCancelSignIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineJid</sender>
   <signal>returnPressed()</signal>
   <receiver>ChatWindow</receiver>
   <slot>slotSignIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linePassword</sender>
   <signal>returnPressed()</signal>
   <receiver>ChatWindow</receiver>
   <slot>slotSignIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabs</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>ChatWindow</receiver>
   <slot>slotTabCloseRequested(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabs</sender>
   <signal>currentChanged(int)</signal>
   <receiver>ChatWindow</receiver>
   <slot>slotCurrentTabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAddTab</sender>
   <signal>clicked()</signal>
   <receiver>ChatWindow</receiver>
   <slot>slotAddTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSignIn()</slot>
  <slot>slotCancelSignIn()</slot>
  <slot>slotTabCloseRequested(int)</slot>
  <slot>slotCurrentTabChanged(int)</slot>
  <slot>slotAddTab()</slot>
 </slots>
</ui>
